src/Repository.js

```diff
@@ -19,11 +20,30 @@
 
   LifeGamification.repository.updateSkills = function (skills) {
     return new Promise((resolve, reject) => {
-      chrome.storage.sync.clear(function() {
-        chrome.storage.sync.set({skillsCollectionId: skills}, function() {
-          resolve(skills);
-        });
+      chrome.storage.sync.set({skillsCollectionId: skills}, function() {
+        resolve(skills);
       });
     });
   }
+
+  LifeGamification.repository.setWork = function (work) {
+    return new Promise((resolve, reject) => {
+      chrome.storage.sync.set({workId: work}, resolve);
```

Comment:
What if a user wants to have different timers? For example one for sitting straight and another for doing math.